src/promptflow/tests/sdk_cli_azure_test/recording_utilities/bases.py

```diff
@@ -230,6 +256,17 @@ def _drop_duplicate_recordings(self) -> None:
         return
 
     def _custom_request_path_matcher(self, r1: Request, r2: Request) -> bool:
+        # NOTE: orders of below conditions matter, please modify with caution
+        # in run download scenario, observed below wired path: https://<xxx>/https://<yyy>/<remaining>
```

Comment:
Is this the same issue with above github issue?

Reply:
no, this is the issue we synced before, I wonder it's a blob client behavior: get the path somewhere (maybe PFS), then use it to download